build.gradle

```diff
@@ -17,11 +17,15 @@ plugins {
 // Define the main class for the application
 mainClassName = 'com.scylladb.java.StreamsAdapterDemo'
 
+apply plugin: 'java'
+sourceCompatibility = 1.8
```

Comment:
are reason why we are pinning to this version ?

Reply:
Most our projects are on 1.8 source level. I kept it there to decrease chances it foobars in any build or whatnot.